### DISCUSSION TITLE
Kata 3 Condicion no satisface todos los valores

### DISCUSSION THREAD

**Discussion by dannramirez** [LaunchX-InnovaccionVirtual/CursoIntroPython#107] [Q&A] [answered]
La redacción del problema solicita que la condición a cumplir es que la velocidad del asteroide sea superior a 25km/hora y superior a 25m para poder mandar un mensaje de alerta pero si los valores son 25 y 25 no se cumple y se mandaria por default el valor del else de que no pasa nada.

<img width="1154" alt="Captura de Pantalla 2022-02-10 a la(s) 20 30 47" src="https://user-images.githubusercontent.com/68350246/153529040-fd7732f2-29d1-4ece-bd54-ab8cbe81cf43.png">

por lo que tenemos que agregar una condicion adicional para poder contemplar cuando la velocidad sea de 25km/hr y de un tamaño de 25m.

O solo se debe cumplir el requerimiento solicitado aun cuando no se satisface con esos valores?

Gracias

**Comment by dannramirez** [comment, marked as answer]
Cuando el valor de la velocidad es de 25km/hr y un tamaño de 25m encontre que debe de ir en un mensaje para mirar el cielo!
Espero a alguien le resulte util esta observación

<img width="1154" alt="Captura de Pantalla 2022-02-10 a la(s) 20 37 51" src="https://user-images.githubusercontent.com/68350246/153529580-e9160e61-4fad-4f84-801d-e4e7246c5672.png">

**Comment by D4N1ELPAT** [comment]
pues yo lo que puse es que como la segunda dice mas que 25 osea 26 osea que los de 25 se quemaran en la atmosfera

**Reply by dannramirez** [reply]
si, asi es como debe de quedar, lo que pasa es que no habia puesto esa condición por lo que a mi me mandaba el error de que el espacio se encontraba en estado normal.
Gracias por reforzar la respuesta que obtuve

**Reply by RodolfoMorquecho** [reply]
Hola, me tomo el atrevimiento ya que vi que solventaste tu duda.  En los ejercicios dice que esta kata se hace en python pero veo que lo haces en visual, se puede hacer como notebook? perdon si me salgo de tema y entiendo si no lo quieren responder.

**Reply by BarajasAngel** [reply]
Efectivamente si se puede hacer desde notebook, gracias a qué permite escritura y código .

**Reply by RodolfoMorquecho** [reply]
Gracias por responder , aunque me refería mas a que si se podía entregar haciedolo desde un notebook, no a que si visual studio te permitía esa función, ya que en la kata2 lo hice con impresiones de pantalla a la consola y creí que en la kata3 era igual.

**Reply by BarajasAngel** [reply]
Si, las katas se pueden entregar en Notebook, de todos modos en la sesión de hoy viernes especificarán el metodo de entrega sin embargo en uno de los live la comander dijo que de podía usar Notebooks por si gusta revista 👌

**Reply by RodolfoMorquecho** [reply]
Gracias de nuevo, si he estado en todos los lives solo que me surgió esa confusión ahora pero ya me despejaste esa duda.

**Reply by dannramirez** [reply]
Si, la kata 2 es la unica que se entrega con capturas de pantalla, las demas se suben a un repo y de ahi las vamos a entregar, perdon por responder un poco tarde.
Saludos
